Fails to build on `no_std`

<!-- Describe the problem you are experiencing. -->
There are tons of errors when trying to build `palette` with `--no-default-features`.
## How To Reproduce
- Can reproduce on `cargo 1.82.0-nightly (2f738d617 2024-08-13)` and `cargo 1.80.1 (376290515 2024-07-16)`
- Clone this repo
- `cd palette`
- Run `cargo b --no-default-features`
<!-- How can the problem be reproduced? Minimal example programs that demonstrate the problem are preferred! -->

## Expected Outcome
Builds without errors
<!-- How should the example(s) above behave? What is the expected outcome? -->

## Actual Outcome
32 errors. Here is one of them:
```rs
error[E0277]: the trait bound `f32: Round` is not satisfied
   --> palette/src/angle.rs:127:40
    |
127 |                     self - Round::ceil(((self + 180.0) / 360.0) - 1.0) * 360.0
    |                            ----------- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Round` is not implemented for `f32`
    |                            |
    |                            required by a bound introduced by this call
...
179 | impl_angle_float!(f32, f64);
    | --------------------------- in this macro invocation
    |
help: this trait has no implementations, consider adding one
   --> palette/src/num.rs:234:1
    |
234 | pub trait Round {
    | ^^^^^^^^^^^^^^^
    = note: this error originates in the macro `impl_angle_float` (in Nightly builds, run with -Z macro-backtrace for more info)
```
<!-- What happens with the current version? How is it incorrect? Include error messages if there are any. -->

## Additional Details
I am trying to use palette on the `thumbv6m-none-eabi` target. I am trying to use palette to control the onboard RGB LED on a RP2040 Zero.
* Cargo.toml entry: <!-- Include any changes to the features --> no changes
* Rust version(s): <!-- Run `rustc -V` if you are not sure --> `cargo 1.82.0-nightly (2f738d617 2024-08-13)` and `cargo 1.80.1 (376290515 2024-07-16)`
* Target platform: <!-- Either your own OS or the `--target some-platform` argument when cross compiling --> `x86_64-unknown-linux-gnu` and `thumbv6m-none-eabi`

--- a/palette/README.md
+++ b/palette/README.md
@@ -41,7 +41,8 @@ features = ["libm"] # Uses libm instead of std for floating point math
 
 These features are enabled by default:
 
-* `"named"` - Enables color constants, located in the `named` module.
+* `"named"` - Enables color constants, located in the
+  [`named`](https://docs.rs/palette/latest/palette/named/index.html) module.
 * `"std"` - Enables use of the standard library. Also enables `"alloc"`.
 * `"alloc"` - Enables implementations for allocating types, such as `Vec` or `Box`.
 * `"approx"` - Enables approximate comparison using [`approx`].

--- a/palette/src/lib.rs
+++ b/palette/src/lib.rs
@@ -2,6 +2,61 @@
 //! accessible for anyone. It uses the type system to enforce correctness and to
 //! avoid mistakes, such as mixing incompatible color types.
 //!
+//! # Getting Started
+//!
+//! Add the following lines to your `Cargo.toml` file:
+//!
+//! ```toml
+//! [dependencies]
+//! palette = "0.7.6"
+//! ```
+//!
+//! or these lines if you want to opt out of `std`:
+//!
+//! ```toml
+//! [dependencies.palette]
+//! version = "0.7.6"
+//! default-features = false
+//! features = ["libm"] # Uses libm instead of std for floating point math
+//! ```
+//!
+//! ## Cargo Features
+//!
+//! These features are enabled by default:
+//!
+//! * `"named"` - Enables color constants, located in the [`named`] module.
+//! * `"std"` - Enables use of the standard library. Also enables `"alloc"`.
+//! * `"alloc"` - Enables implementations for allocating types, such as `Vec`
+//!   or `Box`.
+//! * `"approx"` - Enables approximate comparison using [`approx`].
+//!
+//! These features are disabled by default:
+//!
+//! * `"serializing"` - Enables color serializing and deserializing using
+//!   [`serde`].
+//! * `"random"` - Enables generating random colors using [`rand`].
+//! * `"libm"` - Uses the [`libm`] floating point math library (for when the
+//!   `std` feature is disabled).
+//! * `"bytemuck"` - Enables casting between plain data types using
+//!   [`bytemuck`].
+//! * `"wide"` - Enables support for using SIMD types from [`wide`].
+//! * `"find-crate"` - Enables derives to find the `palette` crate when it's
+//!   renamed in `Cargo.toml`.
+//!
+//! These features have been deprecated:
+//!
+//! * `"named_from_str"` - Alias for `"named"`, still enabled by default.
+//!   <strike>Enables `named::from_str`, which maps name strings to
+//!   colors.</strike>
+//!
+//! ## Using palette in an embedded environment
+//!
+//! Palette supports `#![no_std]` environments by disabling the `"std"`
+//! feature. It uses [`libm`], via the `"libm"` feature, to provide the
+//! floating-point operations that are typically in `std`, and the `"alloc"`
+//! feature to provide features that use allocating types. However, serializing
+//! with `serde` is not available without the standard library.
+//!
 //! # Where Do I Start?
 //!
 //! The sections below give an overview of how the types in this library work,

